build.gradle

```diff
@@ -99,7 +99,10 @@ sonarqube {
     property "sonar.projectName", "Reform :: bulk-scan-ccd-event-handler-sample-app"
     property "sonar.projectKey", "uk.gov.hmcts.reform:bulk-scan-ccd-event-handler-sample-app"
     property "sonar.coverage.jacoco.xmlReportPaths", jacocoTestReport.reports.xml.destination.path
-    property "sonar.exclusions", "**/config/**"
+    // CaseUpdateDetails mostly duplicates ExceptionRecord.
+    // Introducing inheritance to avoid this duplication would make these classes coupled
+    // though they are intended for different purposes.
```

Comment:
Worth adding a todo to remove the exclusion when ExceptionRecord is gone

Reply:
ok